material-components/material-components-ios
# [BottomNavigation] Add accessibilityHint support in the tab bar item view.

Propagate a `MDCBottomNavigationItemView`'s accessibilityHint to its `button`.

Closes #7290 

Steps for manually testing:
1. Turn on Voice Over.
2. Open the "Bottom Navigation" example.
3. Make Voice Over focus on the "Reader" tab (4th item from left).
4. If the Reader tab is unselected, ensure that it reads "Reader, 88, Tab, 4 of 5 - Shows the reader screen." (Note: there is a delay between "4 of 5" and "Shows the reader." This is expected and consistent with Apple's `UITabBar`.)
5. Select the Reader tab. Ensure it reads "Selected, Reader, 88, Tab, 4 of 5." (Note: it does not read the hint here, which is expected and consistent with Apple's `UITabBar` behavior.)
